requirements/dev-requirements.txt

```diff
@@ -10,6 +10,8 @@ alembic==1.4.3
     # via -r base-requirements.in
 amqp==2.6.1
     # via kombu
+appnope==0.1.3
+    # via ipython
```

Comment:
`appnope` again! I'd recommend scripting something to make `make requirements` end by removing `appnope` lines like this from the .txt files

Reply:
aghhhhh

Reply:
yeah i'm going to do this now

Reply:
@dannyroberts https://github.com/dimagi/commcare-hq/pull/31400